M/MSTM/build_tarballs.jl

```diff
@@ -1,7 +1,7 @@
 using BinaryBuilder, Pkg
 
 name = "MSTM"
-version = v"4.0.0"
+version = v"4.0.1"
```

Comment:
We tend to keep the same version number as upstream, if possible

Reply:
Done